src/components/page/ResultPage.js

```diff
@@ -147,21 +147,20 @@ class Result extends React.Component {
                     {result.Description}
                 </Description>
 
-                <GradationText fontSize={0.7}>
+                <GradationText fontSize={1}>
                     Flex your level
                 </GradationText>
 
                 <ShareLink />
 
-                <GradationButton color="white" onClick={()=>window.location.href = '/'}>
-                    Start a new quiz
-                </GradationButton>
+                <GradationButton 
+                    onClick={()=>window.location.href = '/'}
+                    text={"Start a new quiz"}/>
```

Comment:
It seems way better!
text={"Start a new quiz"} />
please make space above at the end of the tag